php/.gitignore

```diff
@@ -8,3 +8,4 @@ composer.lock
 .deps
 .compared
 .tested
+.built
```

Comment:
this `.built` file is an empty file used by Makefile as a marker. This avoids committing it by mistake when doing the commit without running `make mostlyclean` before (which would delete that file)